_includes/head.html

```diff
@@ -16,4 +16,22 @@
   <link rel="manifest" href="{{ site.url }}/manifest.json">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ site.url }}/feed.xml">
   <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
-</head>
+  {% if page.layout == 'permalink' %}
+  <meta name="twitter:card" content="summary">
+  {% else %}
+  <meta name="twitter:card" content="summary_large_image">
+  {% endif %}
+  <meta name="twitter:title" content="{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}">
+  <meta name="twitter:site" content="@pwastats">
+  <meta name="twitter:creator" content="@cloudfour">
+  {% if page.title %}
+    <meta property="og:site_name" content="{{ site.title }}"/>
+  {% endif %}
+  {% if page.layout == 'permalink' %}
+  <meta property="og:title" content="{% if page.title %}{{ page.title }}{% endif %}" />
```

Comment:
I'm not claiming to be _doing it right_ - this is what I am seeing: 

![twitter](https://user-images.githubusercontent.com/1427548/29538675-b3e977aa-867b-11e7-9279-a8e17156856a.gif)

The error message I see when the validator fails after removing `twitter:title` is:

>>> INFO:  Page fetched successfully
INFO:  14 metatags were found
INFO:  twitter:card = summary_large_image tag found
ERROR: Possibly invalid value (if specified) (og:title)
ERROR: Possibly invalid value (if specified) (twitter:text:title)
ERROR: Possibly invalid value (if specified) (twitter:title)